test/functional/wallet_createwallet.py

```diff
@@ -96,5 +96,28 @@ def run_test(self):
         assert_raises_rpc_error(-4, "Error: This wallet has no available keys", w5.getnewaddress)
         assert_raises_rpc_error(-4, "Error: This wallet has no available keys", w5.getrawchangeaddress)
 
+        self.log.info('New blank and encrypted wallets can be created')
+        self.nodes[0].createwallet(wallet_name='wblank', disable_private_keys=False, blank=True, passphrase='thisisapassphrase')
+        wblank = node.get_wallet_rpc('wblank')
+        assert_raises_rpc_error(-13, "Error: Please enter the wallet passphrase with walletpassphrase first.", wblank.signmessage, "needanargument", "test")
```

Comment:
micronit: I always prefer named arguments in tests, so there's no ambiguity about what the RPC method call should be. In that case it'd be `...wblank.signmessage, address="fakeaddress", message="test"` here.

EDIT: `assert_raises_rpc_error()` itself takes an argument called `message`, so this doesn't work. We should fix that (in a different PR) by making `assert_raises_rpc_error`'s argument something like `_error_message` so it doesn't collide with any RPC arguments.

Reply:
Or use a lambda to bind the named args?